src/hooks/useWorkletStateMachine/runOnUISync/index.ts

```diff
@@ -0,0 +1,7 @@
+// runOnUISync crashes on web not because browsers lack a UI thread concept,
```

Comment:
https://github.com/software-mansion/react-native-reanimated/blob/4efcaca45927a4a74f0b1a4c1a2bef8b08145b2f/packages/react-native-worklets/src/threads.ts#L25

This suggests that `runOnUISync` (as `execcuteOnUIRuntimeSync`) is not supported on the web